packages/devtools-reps/src/reps/rep-utils.js

```diff
@@ -387,6 +387,19 @@ function isURL(token) {
   }
 }
 
+/**
+ * Gets the ellipsis element.
+ *
+ * @return React.Element
+ */
+function getMoreEllipsisElement() {
```

Comment:
Thanks for putting this in a util function, the code looks better.
Maybe we could even have the element in a constant ? Like

```js
const ellipsisElement = span({
  title: `more${ELLIPSIS}`
}, ELLIPSIS);

module.exports = {
  ...,
  ellipsisElement
}
```

Since the element is always the same, we save a bit of time by not having to re-create it for each Reps that need it.

Reply:
I thought of that too actually but I went with playing safe! :)

Reply:
I think that we should keep `key` and `className`, since it's often used as an array item and its styling is pretty standard across the codebase, resulting to:
```javascript
const ellipsisElement = span({
    key: "more",
    className: "more-ellipsis",
    title: `more${ELLIPSIS}`
}, ELLIPSIS);
```